frontend/package.json

```diff
@@ -50,5 +50,17 @@
     "jest-date-mock": "^1.0.3",
     "prettier": "^1.13.5",
     "redux-mock-store": "^1.5.3"
+  },
+  "browserslist": {
```

Comment:
Any reason why you added this? We can't guarantee what browser the users are going to use, so best not to have it

Reply:
When we updated react scripts, it asks to add browserslist.

Reply:
Ah yeah that's annoying! We do this on the platform so try this `"browserslist": "last 2 versions, ie 10"`